docs/trento-server.md

```diff
@@ -116,13 +110,13 @@ Now you can connect to the web server via `http://localhost` and point the agent
 Use a different container image (e.g. the `rolling` one):
 
 ```
-helm install trento . --set trento-web.image.tag="rolling" --set trento-runner.image.tag="rolling" --set-file trento-runner.privateKey=id_rsa_runner
+helm install trento . --set trento-web.image.tag="rolling" --set trento-wanda.image.tag="rolling"
 ```
 
 Use a different container registry:
 
 ```
-helm install trento . --set trento-web.image.repository="ghcr.io/myrepo/trento-web" --set trento-runner.image.repository="ghcr.io/myrepo/trento-runner" --set-file trento-runner.privateKey=id_rsa_runner
-```
+helm install trento . --set trento-web.image.repository="ghcr.io/myrepo/trento-web" --set trento-wanda.image.repository="ghcr.io/myrepo/trento-wanda"
 
 Please refer to the the subcharts `values.yaml` for an advanced usage.
+```
```

Comment:
This code block looks wrong, doesn't it?
I think it was ok beforehand, just removing the privateKey thing

Reply:
thanks